### ISSUE TITLE
Immediate Actionables (Critical for New Year)

### ISSUE THREAD

**Issue by lbxa** [lbxa/o#81]
### Immediate Actionables (Critical for New Year)

1. **User Management Features**  
   - [x] #82
     - Names  
     - Usernames  
     - Profile pictures (fallback: use initials inside coloured circles for simplicity)  
   - [ ] Implement account deletion functionality.  

2. **Community Management Features**  
   - [x] #83
   - [x] #101

3. **Streaks and Homepage Sync**  
   - [x] #84
   - [x] #94
   - [x] #85

4. **Challenge Details**  
   - [x] #86
   - [x] #106
   - [x] #93

5. **Tweaks**  
   - [x] #87
   - [x] #89
   - [x] #90
   - [x] #105
   - [x] #91

**Comment by JamieGom** [comment]
Makes sense. However, what about editing the challenge name / description as people tend to make typos and realise later on? Not urgent though

**Comment by lbxa** [comment]
> Makes sense. However, what about the option to only being able to edit the challenge name / description as people tend to make typos and realise later on? Not urgent though

Good point, agreed. I'll add that in.

**Comment by JamieGom** [comment]
@lbxa When you work on 'Sync homepage data dynamically with challenges. #85' check out my comment because you may not see it if you don't click on it :)

**Comment by lbxa** [comment]
I've removed the notification tweak from this list so it can be re-prioritised with the other tickets @JamieGom :D
